### ISSUE TITLE
What is meant with secure and asserting the identity of the OOB

### ISSUE THREAD

**Issue by gloinul** [EricssonResearch/blind-cache-draft#21]
In the section on OOB Cache Discovery the following is written:

The discovery mechanism MUST be secure in the sense that the client is assumed to assert the identity of the oob cache.

What is meant with this. What type of identity assertion is the client expected to do. I assume it is to the Origin that one makes an identity assertion.

**Comment by gaperik** [comment]
Good catch- with the latest change, asserting the identify was moved from the client to the origin. The original design was to assume that the discovery process is security by itself, meaning the client could rely on the discovery process to provide a valid URL to a oob cache. I guess mDNS with DNSSEC could be an idea? But with the origin verifying the oob caches identity, the risk of the client proposing a malicious cache need to be taken into account.
